plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/completion/BazelrcCompletionContributor.kt

```diff
@@ -0,0 +1,106 @@
+package org.jetbrains.bazel.languages.bazelrc.completion
+
+import com.intellij.codeInsight.completion.CompletionContributor
+import com.intellij.codeInsight.completion.CompletionParameters
+import com.intellij.codeInsight.completion.CompletionProvider
+import com.intellij.codeInsight.completion.CompletionResultSet
+import com.intellij.codeInsight.completion.CompletionType
+import com.intellij.codeInsight.lookup.LookupElement
+import com.intellij.codeInsight.lookup.LookupElementBuilder
+import com.intellij.patterns.PlatformPatterns.psiElement
+import com.intellij.util.ProcessingContext
+import org.jetbrains.bazel.assets.BazelPluginIcons
+import org.jetbrains.bazel.languages.bazelrc.BazelrcLanguage
+import org.jetbrains.bazel.languages.bazelrc.elements.BazelrcTokenTypes
+import org.jetbrains.bazel.languages.bazelrc.psi.BazelrcFile
+
+class BazelrcCompletionContributor : CompletionContributor() {
+  init {
+    extend(CompletionType.BASIC, BazelCommandCompletionProvider.psiElementPattern, BazelCommandCompletionProvider())
+    extend(CompletionType.BASIC, BazelConfigCompletionProvider.psiElementPattern, BazelConfigCompletionProvider())
+  }
+}
+
+class BazelCommandCompletionProvider : CompletionProvider<CompletionParameters>() {
+  companion object {
+    val psiElementPattern =
+      psiElement()
+        .withLanguage(BazelrcLanguage)
+        .andOr(
+          psiElement(BazelrcTokenTypes.COMMAND),
+          psiElement().beforeLeaf(psiElement(BazelrcTokenTypes.COMMAND)),
+        )
+
+    // TODO(mtoader): Find a nicer way of building this, and the "set of correct flag with type / value for command" ?
```

Comment:
can you please create a ticket on our youtrack for that?

Reply:
Filed [BAZEL-1195](https://youtrack.jetbrains.com/issue/BAZEL-1195/Need-a-nice-way-to-know-what-flags-are-correct-for-a-particular-bazel-version)